sdk/servicebus/azure-servicebus/doc/azure.servicebus.rst

```diff
@@ -5,7 +5,7 @@ azure.servicebus package
    :members:
    :undoc-members:
    :inherited-members:
-   :exclude-members: to_bytes, from_bytes
+   :exclude-members: to_bytes, from_bytes, as_integer_ratio, bit_count, bit_length, conjugate, imag, numerator, real, capitalize, casefold, center, count, encode, endswith, startswith, expandtabs, find, format, format_map, index, isalpha, isalnum, isascii, isdecimal, isdigit, isidentifier, islower, isnumeric, isprintable, isspace, istitle, isupper, join, ljust, lower, lstrip, maketrans, partition, removeprefix, removesuffix, replace, rfind, rindex, rjust, rpartition, rsplit, rstrip, split, splitlines, strip, swapcase, title, translate, upper, zfill, denominator
```

Comment:
thanks l0l! and for everything except PEEK_LOCK and RECEIVE_AND_DELETE here? https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/latest/azure.servicebus.html#azure.servicebus.ServiceBusReceiveMode